code/src/declaration.d.ts

```diff
@@ -0,0 +1,19 @@
+declare module "*.svg" {
+  const content: string;
+  export default content;
+}
+
+declare module "*.png" {
+  const content: string;
+  export default content;
+}
+
+declare module "*.jpg" {
+  const content: string;
+  export default content;
+}
+
+declare module "*.jpeg" {
+  const content: string;
+  export default content;
+}

```

Comment:
Is there support for `.webp`? It is rather popular and one of recommended format for images on websites

Reply:
> Is there support for `.webp`? It is rather popular and one of recommended format for images on websites

Not sure that it should be here.
We discussed to use `jpeg` and `svg` for icons